### ISSUE TITLE
Date Format Changed in High Sierra

### ISSUE THREAD

**Issue by grokdesigns** [aaronpk/iMessage-Export#6]
It appears that the format for dates has changed in High Sierra. You can edit the following function to fix it. You need the first 9 characters from the new date/timestamp for it to be correct. Change the `date+978307200` part of the query to `substr(date,1,9)+978307200`.
```
function query_messages_since(&$db, $timestamp) {
  return $db->query('SELECT message.ROWID, date+978307200 AS date, 
    message.text, is_from_me, handle.id AS contact
  FROM message
  LEFT JOIN handle ON message.handle_id = handle.ROWID
  WHERE cache_roomnames IS NULL
    AND date+978307200 > ' . $timestamp . '
  ORDER BY date
  ');
}
```

**Comment by rahulsindc** [comment]
Is there a fix planned here (or conversely, is there a client-side fix? I'm a little confused about this format:
"Timestamp": 546062408473794200,
      "Date": "17304030840-07-10",
      "Time": "23:42:56",

Though I suppose one could always work backwards from Unix Time

**Comment by grokdesigns** [comment]
@rahulsindc You can make the change on lines 47 and 52 as indicated in my initial post. Replace `date+978307200` with `substr(date,1,9)+978307200` on both lines.

**Comment by aaronpk** [comment]
Fixed in #8, thanks!
